analyzers/its/expected/akka.net/Akka.Streams--netstandard2.0-S2259.json

```diff
@@ -0,0 +1,82 @@
+{
+"issues":  [
+{
+"id":  "S2259",
+"message":  "'graph' is null on at least one execution path.",
+"location":  {
+"uri":  "sources\akka.net\src\core\Akka.Streams\Dsl\Flow.cs",
+"region":  {
+"startLine":  477,
+"startColumn":  76,
+"endLine":  477,
+"endColumn":  81
+}
+}
+},
```

Comment:
False Positive. `graph` is not checked for null and thus no constraint should be learned.
https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/akka.net/src/core/Akka.Streams/Dsl/Flow.cs#L477